packages/yoga/src/Feedback/web/Feedback.jsx

```diff
@@ -75,6 +78,9 @@ function Feedback({ variant, title, description, children, center, ...props }) {
             {description}
           </Text.Body1>
         </TextContainer>
+
+        {captionElement ? <Box mt="medium">{captionElement}</Box> : null}
```

Comment:
Shouldn't this content be text?

Reply:
This content is a `text`, `captionElement` is a `Text.Body1`

This happens in line 50 where we have:
`if (isChildFromComponent(child, Caption)) captionElement = child;`